samples/drumthumper/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.google.oboe.sample.drumthumper">
+    package="com.plausible.drumthumper">
```

Comment:
plausible.com website currently does not exist.
Are you planning to at least create a placeholder?

Reply:
Is it required? (I didn't find anything for "mobilier.com")
Does Google provide an easy mechanism or would I have to buy the URL and set it up?

Reply:
mobilEEr.com
You have, to at least, own the domain. That is how they guarantee the uniqueness of the package name. It is a reverse domain name. I suspect they look for a website when screening for malware. But I am not sure? Do you have a domain and website?

Reply:
Done. com.mobileer used.

Reply:
I don't know if you can use my domain from your account.

Reply:
Here are some handy looking docs:
https://developer.android.com/studio/publish